site/content/docs/manifest/environment.en.md

```diff
@@ -0,0 +1,177 @@
+              public:
+                - id: 'subnet-11111'
+                - id: 'subnet-22222'
+              private:
+                - id: 'subnet-33333'
+                - id: 'subnet-44444'
+        ```
+
+    === "Configured VPC"
+
+        ```yaml
+        name: qa
+        type: Environment
+        network:
+          vpc:
+            cidr: '10.0.0.0/16'
+            subnets:
+              public:
+                - cidr: '10.0.0.0/24'
+                  az: 'us-east-2a'
+                - cidr: '10.0.1.0/24'
+                  az: 'us-east-2b'
+              private:
+                - cidr: '10.0.3.0/24'
+                  az: 'us-east-2a'
+                - cidr: '10.0.4.0/24'
+                  az: 'us-east-2b'
+        ```
+
+    === "With public certificates"
+
+        ```yaml
+        name: prod-pdx
+        type: Environment
+        http:
+          public: # Apply an existing certificate to your public load balancer.
+            certificates:
+              - arn:aws:acm:${AWS_REGION}:${AWS_ACCOUNT_ID}:certificate/13245665-cv8f-adf3-j7gd-adf876af95
+        ```
+
+    === "Private"
+
+        ```yaml
+        name: onprem
+        type: Environment
+        network:
+          vpc:
+            id: 'vpc-12345'
+            subnets:
+              private:
+                - id: 'subnet-11111'
+                - id: 'subnet-22222'
+                - id: 'subnet-33333'
+                - id: 'subnet-44444'
+        http:
+          private: # Apply an existing certificate to your public load balancer.
+            certificates:
+              - arn:aws:acm:${AWS_REGION}:${AWS_ACCOUNT_ID}:certificate/13245665-cv8f-adf3-j7gd-adf876af95
+            subnets: ['subnet-11111', 'subnet-22222']
+        ```
+
+<a id="name" href="#name" class="field">`name`</a> <span class="type">String</span>  
+The name of your environment.
+
+<div class="separator"></div>
+
+<a id="type" href="#type" class="field">`type`</a> <span class="type">String</span>  
+Must be set to `'Environment'`.
+
+<div class="separator"></div>
+
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>  
+The network section contains parameters for importing an existing VPC or configuring the Copilot generated VPC.
+
+<span class="parent-field">network.</span><a id="network-vpc" href="#network-vpc" class="field">`vpc`</a> <span class="type">Map</span>  
+The vpc section contains parameters to configure CIDR settings and subnets.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-id" href="#network-vpc-id" class="field">`id`</a> <span class="type">String</span>    
+The ID of the VPC to import. This field is mutually exclusive with `cidr`.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-cidr" href="#network-vpc-cidr" class="field">`cidr`</a> <span class="type">String</span>    
+An IPv4 CIDR block to associate with the Copilot-generated VPC. This field is mutually exclusive with `id`.
```

Comment:
```suggestion
An IPv4 CIDR block to associate with the Copilot generated VPC. This field is mutually exclusive with `id`.
```
to match `network` (idc which way, just wanna be consistent)

Reply:
Done! fixed `network` instead 😂